code/game/objects/items/weapons/kitchen.dm

```diff
@@ -138,6 +138,13 @@
 	icon_state = "render"
 	w_class = WEIGHT_CLASS_NORMAL
 
+/obj/item/kitchen/knife/shiv
+	name = "glass shiv"
+	desc = "A haphazardly constructed blade, fashioned from a glass shard wrapped in cloth and tied with wire."
```

Comment:
Might just be me but I'd prefer some flavour text over how to make the thing, as I don't need to know that if I already made the damn thing

Reply:
Will change this to be more interesting when I’m back at my PC, thanks